lib/YaST/Bootloader/BootCodeOptionsTab.pm

```diff
@@ -3,10 +3,10 @@
 # Copyright 2021 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
-# Summary: The class introduces all accessing methods for Bootloader
+# Summary: The class introduces all accessing methods for Boot code options tab in bootloader module
```

Comment:
The question is if we will cover more of that module in the installer, or in the desktop module ? Back when I implemented the bootcode options module I had the misunderstanding that we wanted to merge everything under Yast.

Reply:
the modules will come later in the roadmap and now the code with more quality and more homogeneous is in Installation, we don't know yet, because this module is exactly the same in both, for testing error message is great in the installer, the module doesn't help the user at all when the installer give you the pre-selected options that makes sense and error message in Installation Settings, in the installed system you are on your own, but on the other hand for the other tabs, kernel or change the theme, that are more suitable for the installed system. Feel free to leave it as it is, it is just about code organization.